.github/workflows/macos.yml

```diff
@@ -19,17 +19,21 @@ on:
         type: string
       # System configuration
       osversion_xcodeversion_toxenv_tuples:
+        # As of 2024-02, "runs-on: macos-latest" is macos-12.
+        # and "runs-on: macos-14" selects the new M1 runners.
+        # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
         description: 'Stringified JSON object'
         default: >-
-          [["latest", "",             "homebrew-macos-usrlocal-minimal"],
-           ["latest", "",             "homebrew-macos-usrlocal-standard"],
-           ["11",     "xcode_13.2.1", "homebrew-macos-usrlocal-minimal"],
+          [["11",     "xcode_13.2.1", "homebrew-macos-usrlocal-minimal"],
```

Comment:
To keep the overall execution time constant (to not increase the priority of the portability ci runs over the PR runs even more), please remove macos-11 (it's past its end of life anyway)

Reply:
> To keep the overall execution time constant

That's not a relevant criterion. These platforms are tested because we want them tested.

Reply:
> That's not a relevant criterion. 

For me it's a very relevant criterion.

Reply:
I'll not be negotiating with you how I run the portability CI.

For reference: https://github.com/sagemath/sage/pull/36726#issuecomment-1847870000, https://github.com/sagemath/sage/pull/36726#issuecomment-1872331240